.circleci/config.yml

```diff
@@ -41,35 +41,54 @@ workflows:
           context: horizon
           project_id: 38
 
-      - artsy-remote-docker/build:
+      - artsy-remote-docker/buildkit-build:
           <<: *not_staging_or_release
           context: hokusai
-          name: build
+          executor: hokusai/beta
```

Comment:
This in Force' case is for testing [Hokusai beta release](https://github.com/artsy/hokusai#beta-release). Most projects use [Hokusai canonical release](https://github.com/artsy/orbs/blob/7c26a69a6b560d6acb4b31fb267244d89ea42d6a/src/hokusai/hokusai.yml#L10). Can keep this if we want Horizon to also be a Hokusai-beta tester.

Reply:
Good catch. I'll switch over to the canonical release. I don't think Horizon gets deployed enough to make it particularly useful as a canary, and could just confuse devs in the event it does introduce some breaking change.

Reply:
Switched in [5d71a00](https://github.com/artsy/horizon/pull/580/commits/5d71a000979b343c3310b15c2b69e831927b0912)

Reply:
Okay cool. Then the `executor` spec here can be omitted altogether (Orb defaults to canonical).

Reply:
my fault, `executor` is required at one place - [the `test` job definition](https://github.com/artsy/horizon/blob/400cd47b2779dbe9b9fcc68dcbf865abbb5872b4/.circleci/config.yml#L10C1-L11C1).